app/src/main/java/org/p2p/wallet/user/repository/UserTokensRemoteRepository.kt

```diff
@@ -32,9 +32,10 @@ class UserTokensRemoteRepository(
     private suspend fun mapAccountsToTokens(publicKey: PublicKey, accounts: List<Account>): List<Token.Active> {
         val tokens = accounts.mapNotNull { token ->
             val mintAddress = token.account.data.parsed.info.mint
+            val programId = token.account.owner
 
             if (mintAddress == Constants.WRAPPED_SOL_MINT) {
-                // Hiding Wrapped Sol account because we are adding native SOL lower
+                // Hiding Wrapped Sol account because we are adding native SOL below
```

Comment:
😁

Reply:
my hands were itching to fix that from the beginning, but had no reason to change this code